### ISSUE TITLE
Decide on Community Group name

### ISSUE THREAD

**Issue by brianelton** [brianelton/cg-charter#1]

**Comment by brianelton** [comment]
First ChatGPT ideas:

- AccessibleAdvancers Community Coalition
- InclusiveInitiators Network
- AccessMobilizers Collective

:)

**Comment by brianelton** [comment]
Some actual suggestions:

- Expanding WAI Community Group
- Advancing WAI Community Group

**Comment by bakkenb** [comment]
I like "Advancing WAI Community Group" so far.

**Comment by bakkenb** [comment]
Here is my conversation with AI...

> [me]
> What is a good name for a group that does the following...
> "To engage and empower accessibility professionals to advance the W3C WAI mission through community driven initiatives.
> 
> This could include:
> - Creating new outreach efforts to highlight content on the WAI website
> - Developing external community group resources that complement the existing resources on the WAI website
> - Acting as a resource for W3C Working Groups"
> 
> [AI]
> How about "AccessAdvancers"? It embodies the idea of advancing accessibility initiatives while also emphasizing the proactive and empowering nature of the group.
> 
> [me]
> Try again
> 
> [AI]
> How about "AccessAdvancers Collective" or "W3C WAI Champions"? These names convey a sense of proactive engagement and collaboration in advancing accessibility initiatives aligned with the W3C WAI mission.
> 
> [me]
> would like to use the word accessibility in the name
> 
> [AI]
> How about "Accessibility Allies Network"? This name emphasizes the collaborative aspect of the group's mission to support accessibility initiatives and professionals, while also highlighting the focus on accessibility itself.

So what about "Accessibility Allies Community Group" as an option?
I can keep pestering it for more ideas when we detail out more information to use as a prompt.

**Comment by kakinney** [comment]
AI gave me these:

- AccessAdvocates: A group that champions accessibility and advocates for positive change.
- InclusiveTrailblazers: Pioneers in creating accessible content and resources.
- WAIConnect: Bringing together accessibility professionals to collaborate and empower.
- WebReach: Reaching out to enhance web accessibility for all.
- CommunityA11y: Fostering community-driven accessibility initiatives.


The only one I really like is WAIConnect Community Group on this list.  

That said, I do like Advancing WAI Community Group.  @brianelton 

@bakkenb I feel like Accessibility Allies is a mouth full that I already trip over trying to say.  I'm thinking about referencing it in conversations at a conference and having it be something easy and  memorable that people will go join.  Or when we're talking about it what is a name that could have a short reference that works.  I don't think we want to be the AA Community Group.

**Comment by brianelton** [comment]
I think the only reason I don't like WAIConnect is that it's too close to WAI Engage.

(AA Community Group... lol)

**Comment by bakkenb** [comment]
LOL...
If we had the AA Community Group we could create a resources called "The 12 Step Plan: Full accessibility in 12 steps" and we could provide accountability partners. 
Just kidding. Good point. 

I'll keep thinking, and I really like the point of something simple to say (in public) and easy for people to remember.

**Comment by kakinney** [comment]
Yes, I know WAI Connect is close to the old name, I’ll keep thinking too.  I think easy to remember, and say is a good idea.  Will help us with recognition, I hope.  

And you both make me laugh. 😂

**Comment by bakkenb** [comment]
- Enhancing Accessibility Resources Community Group
- Enhancing WAI Resources Community Group
- Enhancing Accessibility Education Community Group

**Comment by kakinney** [comment]
We have landed on the following as some names:

- Enhancing Accessibility Resources Community Group
- Enhancing Accessibility Education Community Group
- Advancing Accessibility Resources Community Group

We collectively decided not to use WAI in the name of this new community group.

**Comment by bakkenb** [comment]
Just some of my thoughts on the above names...

> Enhancing Accessibility Resources Community Group

I like this, but to me it seems like it might be only about "enhancing" existing resources, which seems limiting to what the group could/would be actually doing.

---------------------
> Enhancing Accessibility Education Community Group

I am a little worried that using the word "education" makes it seem like the group is going to focus on some type of program for learning about accessibility. It loses the "self guidance resources" vibe.

---------------------
> Advancing Accessibility Resources Community Group

Currently, I prefer this one the most because it is more about advancing resources. To me that speaks to making them better, creating more, communicating/sharing about them, and educating with them.
